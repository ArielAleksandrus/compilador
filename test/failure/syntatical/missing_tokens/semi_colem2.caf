int soma(int a, int b){
	int c = a + b;
	retorne c
}
programa{}