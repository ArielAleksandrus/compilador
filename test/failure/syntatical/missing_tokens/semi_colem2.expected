ERROR => Line 3, col 10: Unexpected end of expression
